packages/webpack/mixins/config/mixin.core.js

```diff
@@ -7,12 +7,13 @@ const { sync } = require('mixinable');
 const { Mixin } = require('hops-mixin');
 const { internal: bootstrap } = require('hops-bootstrap');
 const SpeedMeasurePlugin = require('speed-measure-webpack-plugin');
+const { inspect } = require('util');
 
 const { sequence, callable } = sync;
 const { validate, invariant } = bootstrap;
 
 const debugConfig = (target, config) =>
-  debug(`hops:webpack:config:${target}`)(config);
+  debug(`hops:webpack:config:${target}`)(inspect(config, false, null));
```

Comment:
This seems like a leftover from a debugging session. Please revert.
Also, for the future: You can use `DEBUG_DEPTH` instead: https://www.npmjs.com/package/debug#environment-variables